task/python-boto3/0.1/python-boto3.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: python-boto3
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
```

Comment:
`0.12.1` is quite old now, by any chance can we have some latest number here?

Reply:
done